src/features/content-answers/ContentAnswersModalFooter.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../styles/variables';
```

Comment:
it looks like styles for the modal align to a grid so you could utilize `$bdl-grid-unit`

Reply:
ill add a general variable like `$bdl-grid-unit-4` or `$bdl-grid-unit-5` to represent `4 or 5 * $bdl-grid-unit` and replace all the padding:16px / 20px. since there are many places need to be updated